drivers/ad917x/ad9172.c

```diff
@@ -0,0 +1,356 @@
+/***************************************************************************//**
```

Comment:
I would mention in the commit message that there is an API already provided by another group and that should be omitted at the review. Or maybe, if there are any changes done to the API, they should be mentioned as well. 
Also, a link to the datasheet in the commit message is always something nice to have.